app/reports/assessments_report.rb

```diff
@@ -34,9 +38,7 @@ def subject_column(assessment)
   end
 
   def active_students_count(assessment)
-    Student.active.inject(0) do |sum, student|
-      sum + student.student_assessments.where(assessment: assessment).count
-    end
+    assessment.student_assessments.count
```

Comment:
This was doing a lot of extra work, so this cuts the time down to a few seconds.